.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Enhancement request
+about: Elastic-Agent Inputs can't do all the things, but maybe it can do your things.
```

Comment:
Is it supposed to be about inputs here?

```suggestion
about: Elastic-Agent's shipper can't do all the things, but maybe it can do your things.
```

Reply:
Thanks for catching all these, I copied all these templates over without reading them closely enough apparently.